### DISCUSSION TITLE
The iso file keeps getting currupted when I try to flash to the sd card

### DISCUSSION THREAD

**Discussion by Pastel1130** [jayofelony/pwnagotchi#275] [General]
I'm new to this and have tried previous versions of the build. I keep getting an error with BalenaEtcher

**Comment by wpa-2** [comment]
Please read the wiki and use the suggested way in there. 
RPI imager is what you should be using

**Comment by MrBumChinz** [comment]
USE RUFUS BROTHER! SOLVED
